### ISSUE TITLE
service types contain "Sample_" prefix

### ISSUE THREAD

**Issue by dirk-thomas** [ros2/rmw_opensplice#253]
## Bug report

**Required Info:**

- Operating System:
  - Ubuntu Bionic (likely any)
- Installation type:
  - source
- Version or commit hash:
  - current master
- DDS implementation:
  - OpenSplice
- Client library (if applicable):
  - N/A

#### Steps to reproduce issue

```
ros2 run demo_nodes_cpp add_two_ints_server
ros2 service list -t
```

#### Expected behavior

```
/add_two_ints [example_interfaces/AddTwoInts]
...
```

#### Additional information

As a consequence features relying on the service type don't work, e.g. `ros2 lifecycle get`.

**Comment by dirk-thomas** [comment]
This might be a regression of the recently merged #251.

**Comment by dirk-thomas** [comment]
Never mind. Problem between keyboard and chair. I had a non-OpenSplice daemon running which isn't aware of the OpenSplice specific prefix `Sample_` and therefore doesn't filter it...
